.github/workflows/testing.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ["3.7", "3.8", "3.9"]
+        python-version: ["3.6", "3.7", "3.8", "3.9", "3.10", "3.11"]
```

Comment:
I think we can remove 3.6 and 3.7

Reply:
we are still using 3.6 for the LSDC GUI - one of the libraries requires that (albula for viewing diffraction images). fortunately, Shekar has found a way to extract that software into a separate process, so we should be able to release that requirement now.